src/SamplesApp/UITests.Shared/UITests.Shared.projitems

```diff
@@ -3155,6 +3160,9 @@
     <Compile Update="C:\Src\GitHub\nventive\Uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml_Controls\Pivot\PivotSimpleTest.xaml.cs">
       <DependentUpon>PivotSimpleTest.xaml</DependentUpon>
     </Compile>
+    <Compile Update="C:\s\nv.github\uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml_Controls\ListView\ListView_DataContext_Propagation.xaml.cs">
```

Comment:
absolute path

Reply:
The absolute path is caused by a visual studio defect w/shared projects. See https://github.com/nventive/Uno/issues/982#issuecomment-499754459. Until it is resolved we will need to keep an eye out for this and manually fix it up.

Reply:
PR has been updated to use relative paths.